app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,7 @@
 <resources>
     <string name="app_name">Demo - Material Dialog</string>
+    <string name="simplematerialdialog">Simple Material Dialog</string>
```

Comment:
According to standards and naming conventions, we should name resources in snake case. We should add '_' after meaningful words. E.g. here is `animated_material_dialog`.

Reply:
owh, okey, thanks for the info.